src/scene.ts

```diff
@@ -3716,6 +3740,11 @@ export class Scene extends AbstractScene implements IAnimatable {
             this.animate();
         }
 
+        // Navigation

```

Comment:
Instead of having crowd here, maybe we could add an observable that a crowd manager can register to?

Reply:
It works with an observer.